src/Tizen.NUI.Components/Controls/RecyclerView/Item/DefaultLinearItem.cs

```diff
@@ -360,6 +359,15 @@ public override void OnRelayout(Vector2 size, RelayoutContainer container)
             }
         }
 
+        /// <summary>
+        /// Get accessibility name.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override string AccessibilityGetName()
+        {
+            return Label.Text + " " + SubLabel.Text;
```

Comment:
Should it be space between label and sublabel? I noticed that in EFL list items main and additional texts are separated by  comma.

Reply:
Apart from this question I have no doubts towards this patch. I see no regression in functional tests.

Reply:
> Should it be space between label and sublabel? I noticed that in EFL list items main and additional texts are separated by comma.

Good catch. Thank you. It has been updated.